openstudio-common-measures.gemspec

```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'bundler', '>= 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.4.3'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.14'
+  spec.add_dependency 'openstudio-extension', '~> 0.5.1'
+  spec.add_dependency 'openstudio-standards', '~> 0.2.15'
```

Comment:
@DavidGoldwasser  -- this version is greater than the version in openstudio-extension-gem, is that as expected? Extension-gem has `0.2.13`.